scriptsTools/convertAllTagsToLowercase.js

```diff
@@ -0,0 +1,91 @@
+let shell = require('shelljs')
+const fs = require('fs')
+
+const ROUTE_FILES = 'C:/Users/LVV941629/Forks/Edson/githubio_source/content'
```

Comment:
Will ROUTE_FILES work on all devices when the project runs?

Reply:
Good observation, each one must place the path of their project, that will be specified